.circleci/config.yml

```diff
@@ -173,51 +173,60 @@ build_and_deploy: &build_and_deploy
             echo "Pantheon environment name is $PANTHEON_ENV"
 
             # Check if the multidev branch site exists. If not, create it
-            terminus multidev:list --format=list --field=Name portlandor > ./env_list.txt
+            terminus multidev:list --format=list --field=Name $CIRCLE_PROJECT_REPONAME > ./env_list.txt
             if grep -Fxq "$PANTHEON_ENV" ./env_list.txt; then
               echo "Found multidev site"
             else
-              # Create a new Multi-Dev with files and database from Dev.
-              echo "Creating new multidev site. Please wait." && terminus multidev:create portlandor.dev $PANTHEON_ENV
+              # Determine base branch of PR
+              #   - Extract PR number from pull request URL
+              #   - Query GitHub API for PR data and extract base branch from JSON base.ref property
+              [[ $CIRCLE_PULL_REQUEST =~ \/pull\/([0-9]+)$ ]]
+              BASE=$(curl -s https://api.github.com/repos/egovPDX/$CIRCLE_PROJECT_REPONAME/pulls/${BASH_REMATCH[1]} | grep -Pzo '(?s)"base":.*?"ref": "\K[A-Za-z0-9_-]+' | tr -d '\0')
+              if [[ $BASE = "master" ]]; then
+                BASE=dev
+              fi
+              # Create a new Multi-Dev with files and database from site corresponding to PR's base branch.
+              echo "Creating new multidev site cloned from $BASE. Please wait." 
+              terminus multidev:create $CIRCLE_PROJECT_REPONAME.$BASE $PANTHEON_ENV
```

Comment:
This probably is a corner condition: If the base branch multi-dev was deleted manually, the script can fall back on Dev since that's guaranteed to be there. Base branch multi-dev can be checked by running "grep -Fxq "$CIRCLE_PROJECT_REPONAME.$BASE" ./env_list.txt;"

Reply:
I like your out-of-the-box thinking, but if the base branch has been deleted then that automatically means the PR is inoperative since the code can no longer be merged into that selected base branch. So there's no point in accommodating this edge case since you can't merge the code anyway. If that case ever happens, I think it's fine for the build to fail. At that point, the PR would need to be updated with a valid base branch which would trigger a new build that should then succeed.